src/inflection.js

```diff
@@ -136,6 +136,22 @@ class Inflection {
       : value === word
   }
 
+  /**
+   * Check to see if the supplied inflection can disambiguate this one
+   * @param {Inflection} infl Inflection object to be used for disambiguation
+   *                          Must have the obligatoryMatches constraint set
+   */
+  disambiguatedBy (infl) {
+    let matched = true
+    for (let feature of infl.constraints.obligatoryMatches) {
```

Comment:
Since `obligatoryMatches` are set separately, I'm thinking if we can handle situations when `obligatoryMatches` are not set by mistake  gracefully, because that seems to be a thing that is easy to miss. How do we set `obligatoryMatches` usually? Do we rely on a language model for that? Can we set constraints on a first `disambiguatedBy` call if they are not set yet? Maybe we could have a flag that will indicate if this field has been initialized yet (a flag might be an overkill, but do we need to distinguish between situations when there is no obligatory matches or when obligatory matches have not been initialized)? 
Also, what should a function return if no `obligatoryMatches` are set (i.e. there is no information about obligatory matches within an inflection at the moment of comparison). Right now in that case a function will return true. Is this correct from business logic point of view? What do you think?

Reply:
we don't actually set obligatoryMatches on an Inflection in any other cases right now. It's something that was added during some of the inflection table refactoring but didn't get used.  I was piggy backing on it for convenience... you are correct in calling this out.  I need to think about the possible scenarios a bit.